lib/potentialbuild.rb

```diff
@@ -876,7 +870,7 @@ def post_results(compiler, pending)
       end
     end
 
-    github_document = if !@failure.nil?
+    github_document = if @failure.nil?
```

Comment:
OK, oops.  The IF block condition was changed but the actual operations inside did not get reversed.  Bleck.